pkg/runtime/runtime.go

```diff
@@ -733,109 +682,96 @@ func (a *DaprRuntime) isAppSubscribedToBinding(binding string, bindingsList []st
 	return false
 }
 
-func (a *DaprRuntime) initInputBindings(registry bindings_loader.Registry) error {
-	if a.appChannel == nil {
```

Comment:
This check was removed, but this is important: we should not init input bindings if the user doesn't have a server to receive them.

Reply:
yeah, it is moved into the `isAppSubscribedToBinding` which does exactly this check.

Reply:
The original idea was that we can skip all input bindings if there's no app channel configured, and treat it as a log message (warning), explicitly alerting the user that because Dapr doesn't know about any user endpoint, it will not init input bindings.

Reply:
got it, reverted this part